src/hmem_cuda.c

```diff
@@ -76,14 +77,17 @@
 
 struct cuda_attr {
 	int   device_count;
+	bool  p2p_access_supported;
 	bool  use_gdrcopy;
 	bool  use_ipc;
 	void *driver_handle;
 	void *runtime_handle;
 	enum cuda_xfer_setting xfer_setting;
 } cuda_attr = {
-	.driver_handle  = NULL,
-	.runtime_handle = NULL,
+	/* Assume CUDA device(s) are incapable of p2p access by default */
+	.p2p_access_supported = false,
```

Comment:
why not also initialize device_count and the other bools?  do we use p2p_access_supported differently somehow?

Reply:
No real reason - just wanted to initialize `p2p_access_supported` with a default in this case. I opted to leave everything else untouched other than the driver/runtime handles. But I'm fine with initializing all the struct members.

Reply:
I think cuda_attr as defined will be initialized to all 0's anyway.

Reply:
Initialized remaining struct members in latest update